docs/release-notes/highlights-7.0.0.asciidoc

```diff
@@ -94,10 +93,10 @@ image::images/highlights-7.0.0-filters.png[Filters menu]
 [float]
 === Saved objects structure improvements
 
-The <<breaking-changes-7.0-saved-objects,structure of saved objects changed>> so 
-that they are now aware of each other.  
-This means you can now export a dashboard,
-including all visualization dependencies, from the Saved objects page. 
+The {kibana-ref}/breaking-changes-7.0.html#breaking-changes-7.0-saved-objects[structure of saved objects changed],
```

Comment:
I think you can actually leave this one unchanged, since that content exists in the Installation and Upgrade Guide too: https://www.elastic.co/guide/en/elastic-stack/7.0/kibana-breaking-changes.html#breaking-changes-7.0-saved-objects